### ISSUE TITLE
"Premieres in" text is squished

### ISSUE THREAD

**Issue by Discookie** [omarroth/invidious#520]
![image](https://user-images.githubusercontent.com/16914176/57169348-6d0df080-6e06-11e9-87f1-ab21a8399be2.png)

```
<h5 class="pure-g">
                
                    Premieres in 1 hour</h5>
```

**Comment by omarroth** [comment]
Should be fixed with 570715100b893b53d3cad68f6907b49aa60fc071.

**Comment by Discookie** [comment]
They are, thank you!
